test_allowedflare.py

```diff
@@ -0,0 +1,81 @@
+from jwt import encode
+from cryptography.hazmat.primitives.asymmetric.rsa import generate_private_key
+from cryptography.hazmat.backends import default_backend
+from datetime import UTC, datetime
```

Comment:
Why is `datetime` considered third party? Are you running `isort` on this (via Ruff)?

Reply:
I thought ruff format would run isort but something must not be working right.